android-exoplayer/src/main/java/com/brentvatne/exoplayer/MultiViewStateView.kt

```diff
@@ -0,0 +1,201 @@
+package com.brentvatne.exoplayer
+
+import android.annotation.SuppressLint
+import android.graphics.Color
+import android.view.Gravity
+import android.view.View
+import android.view.View.OnClickListener
+import android.view.View.OnFocusChangeListener
+import android.widget.FrameLayout
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.content.res.ResourcesCompat
+import androidx.core.view.isVisible
+import androidx.core.view.setPadding
+import com.brentvatne.react.R
+import com.diceplatform.doris.DorisPlayerOutput
+import com.diceplatform.doris.entity.DorisPlayerEvent
+import com.diceplatform.doris.ui.entity.LabelsTranslation
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.modules.i18nmanager.I18nUtil
+
+@SuppressLint("ViewConstructor")
+class MultiViewStateView(
+    private val tvExoplayerView: ReactTVExoplayerView,
+    labelsTranslation: LabelsTranslation?,
+    focusable: Boolean,
+    private val listener: OnVolumeChangedListener,
+) : FrameLayout(tvExoplayerView.context), OnFocusChangeListener, OnClickListener,
+    DorisPlayerOutput {
+
+    interface OnVolumeChangedListener {
+        fun onRequestMute(view: MultiViewStateView, mute: Boolean)
+    }
+
+    private val isRTL = I18nUtil.getInstance().isRTL(context)
+    private val iconSize = (40 * tvExoplayerView.resources.displayMetrics.density).toInt()
+    private val volumeIcon: ImageView = ImageView(tvExoplayerView.context)
+    private val errorMsgView: TextView = TextView(tvExoplayerView.context)
+    private var stateViewFocusable = focusable
+    private val isMute: Boolean
+        get() = tvExoplayerView.exoDorisPlayerView.isMute
+    val isError: Boolean
+        get() = errorMsgView.isVisible
+
+    init {
+        layoutDirection = if (isRTL) View.LAYOUT_DIRECTION_RTL else View.LAYOUT_DIRECTION_LTR
+        isFocusable = focusable
+        foreground = ResourcesCompat.getDrawable(tvExoplayerView.resources, R.drawable.ic_item_focus_selector, null)
+        // add player view
+        addView(tvExoplayerView)
+
+        // add volume icon
+        resetVolumeIcon()
+        volumeIcon.visibility = if (focusable) View.VISIBLE else View.INVISIBLE
+        volumeIcon.isFocusable = false
+        volumeIcon.onFocusChangeListener = this
+        volumeIcon.setBackgroundResource(R.drawable.ic_multiview_circle_bg)
+        volumeIcon.setOnClickListener(this)
+        volumeIcon.setPadding(12)
```

Comment:
i think we should use dp values, otherwise the padding will be different on hd/full-hd/4k tvs. For the margins as well below.